src/collections/_documentation/platforms/android/index.md

```diff
@@ -303,6 +303,47 @@ SentryAndroid.init(this, options -> {
 });
 ```
 
+### Release Health
+
+Monitor the [health of releases]({%- link _documentation/workflow/releases/index.md -%}#release-health) by observing user adoption, usage of the application, percentage of [crashes]({%- link _documentation/workflow/releases/index.md -%}#release-health-glossary), and [session data]({%- link _documentation/workflow/releases/index.md -%}#release-health-glossary). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+To benefit from the health data provided in the Releases v2, you must use at least 2.1.0 of the Android SDK, and enable the collection of release health metrics in the AndroidManifest.xml file.
```

Comment:
this enables the automatic mode, worth mentioning I guess. if not enabled, one still can `Sentry.startSession()` manually